### ISSUE TITLE
Message "You are currently not logged in. You may want to do so now." leads to error page

### ISSUE THREAD

**Issue by rjollos** [trac-hacks/trac-github#95]
On navigating to a page which the user doesn't have permission to view the following message is displayed: **You are currently not logged in. You may want to do so now.** The phrase **do so** is a link to `/login`. When using trac-github plugin the login path is `/github/login`. It can be seen [here](https://bugs.jqueryui.com/github/login) that clicking on **do so** links to an error page **No handler matched request to /login**.

Was the intent of using `/github/login` as the path to allow trac-github to function side-by-side with Trac's login module?

**Comment by aaugustin** [comment]
`/login` points to username/password authentication on code.djangoproject.com for which I originally developed the plugin. I'm OK with either removing the link and letting the user figure it out -- login links aren't far -- or pointing it to `/github/login`, as you wish.

**Comment by rjollos** [comment]
The message with link is added by [_send_user_error](http://trac.edgewall.org/browser/tags/trac-1.0.9/trac/web/main.py?marks=565,574-575#L542). The only possibility I see is to remove or replace  the link using JavaScript. I opened a ticket [#12332](http://trac.edgewall.org/ticket/12332) to discuss.

**Comment by rjollos** [comment]
Another option would be to have a configuration option `[github]` `use_login_path`, and have it default to `False`. When `True`, the path `/login` would be used for the `IRequestHandler`.

Or we can get the same effect using a variable like `[github]` `login_path_prefix`, and have it default to `/github`.
```
login_path = '/%s/login' % self.login_path_prefix.strip('/')
```

**Comment by rjollos** [comment]
Additional thoughts in [trac:#12332:comment:3](https://trac.edgewall.org/ticket/12332#comment:3).

**Comment by rjollos** [comment]
Proposed fix in #127.
